spk/cloudflared/src/service-setup.sh

```diff
@@ -0,0 +1,18 @@
+TOKEN_FILE="${SYNOPKG_PKGVAR}/token"
```

Comment:
Why not use the official configuration file?

https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/configuration/local-management/configuration-file/

Reply:
https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/configuration/local-management/ingress/

Reply:
Correct, config file is used in locally managed tunnels and the token is used in tunnels managed through the dashboard. As far as I understand, remote management is the recommended set up. The service command is the same that's used in the systemd service created by following the official documentation.  

It seems there are also ways to use a credentials file with remote management, but they involve an interactive setup, passing premade configs, or using the API (related issues https://github.com/cloudflare/cloudflared/issues/645, https://github.com/cloudflare/cloudflared/issues/665).